lib/_tls_wrap.js

```diff
@@ -271,6 +271,14 @@ var proxiedMethods = [
   'setPendingInstances'
 ];
 
+// Proxy HandleWrap, PipeWrap and TCPWrap methods
+proxiedMethods.forEach(function(name) {
+  tls_wrap.TLSWrap.prototype[name] = function methodProxy() {
+    if (this._parent[name])
```

Comment:
Perhaps we could remove it somewhere later, but I better secure ourselves with this for now.